week2/homework/src/index.js

```diff
@@ -1,3 +1,70 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+
+const fs = require('fs');
+
+let fileName = 'toDo.json';
+// console.log(fileName);
+function readFile() {
+    return new Promise(
+        resolve => fs.readFile(
```

Comment:
You should provide the encoding (`'utf8'`) here. That way you don't have to do `data.toString()` below.